### PR TITLE
Tooltips on "reset" and "volume" buttons

### PR DESCRIPTION
**Pull request by brunoblaise**
Hi there  I have added a tooltip in the following buttons

- [x] volume
- [x] reset

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -72,7 +72,7 @@
     </div>
 
     <div class='bar floating top' id='top-bar' hide=false>
-      <div id="volume-slider-box">
+      <div id="volume-slider-box" class="tooltip-1  tooltip" data-tooltip="Volume">
```
Comment (maxwofford) [inline index.html:75]: Had to check the screenshot for figuring out the diff between `tooltip` & `tooltip-1`– what if we call them `tooltip` and `tooltip-e` (tooltip east)?

Comment (brunoblaise) [reply index.html]: It is done and renamed to **tooltip-e**

**Comment by brunoblaise** [comment]
It is done and renamed to `tooltip-e`

**Review by Supsource — COMMENTED** [review]
_/_

**Pull request merged**